permuta/enumeration_strategies/insertion_encodable.py

```diff
@@ -0,0 +1,13 @@
+"""
+Enumeration strategies related to the insertion encoding.
+"""
+
+from permuta.enumeration_strategies.abstract_strategy import \
+    EnumerationStrategy
+from permuta.permutils.insertion_encodable import is_insertion_encodable
+
+
```

Comment:
I think it should instead by a symmetry enumeration strategy, with applies to symmetry being "is_topmost_insertion_encodable"

Reply:
I changed it to ` is_insertion_encodable_maximum` which I guess is what you want

Reply:
Yes exactly